frontend/constants/routes.js

```diff
@@ -11,6 +11,8 @@ const ROUTES = {
   DISCOVERY: '/discovery',
   HISTORY: '/history',
   PROFILE: '/profile',
+
```

Comment:
remove all spaces

Reply:
OK, I'll recover 
  ONBOARDINGCOMPLETE: '/onboarding-complete',
   and add : 
SYSTEMCONFIGURATIONS: '/SystemComfigurations',